pacai/ui/topGraphics.py

```diff
@@ -0,0 +1,600 @@
+import math
+import io
+
+from pacai.core.game import Directions
+from pacai.ui import graphicsUtils
+from pacai.ui import graphicsConstants
+from abc import ABC, abstractmethod
+
+class absPane(ABC):
+    @abstractmethod
+    def drawPane(self):
+        pass
+
+    @abstractmethod
+    def updateScore(self, score):
+        pass
+
+    def toScreen(self, pos, y = None):
+        """
+        Translates a point relative from the bottom left of the info pane.
+        """
+
+        if (y is None):
+            x, y = pos
+        else:
+            x = pos
+
+        x = self.gridSize + x  # Margin
+        y = self.base + y
+        return x, y
+
+    def initializeGhostDistances(self, distances):
+        self.ghostDistanceText = []
+
+        size = 20
+        if self.width < 240:
+            size = 12
+        if self.width < 160:
+            size = 10
+
+        for i, d in enumerate(distances):
+            t = graphicsUtils.text(self.toScreen(self.width / 2 + self.width / 8 * i, 0),
+                    graphicsConstants.GHOST_COLORS[i + 1], d, "Times", size, "bold")
```

Comment:
Spoke with Eriq about fonts. He said finding a font or font checker that would work on any machine can be tricky and that this might be made into a separate issue.